MigrationV3V4/CSharp/Skills/V3SimpleSandwichBot/Sandwich.cs

```diff
@@ -49,7 +49,7 @@ public static IForm<SandwichOrder> BuildForm()
             // When the skill completes (OnCompletion), send an `endOfConversation` activity and include the finished order.
             return new FormBuilder<SandwichOrder>()
                     .Message("Welcome to the simple sandwich order bot! Say 'end' or 'stop' and I'll end the conversation and back to the parent.")
-                    .OnCompletion((context, order) => SkillsHelper.EndSkillConversation(context.Activity as Activity, order))
+                    .OnCompletion((context, order) => ConversationHelper.EndConversation(context.Activity as Activity, order))
```

Comment:
Shouldn't you also call clear conversation state after this? Another option is to just include a call to clear state as part of the EndConversation implementation.

Reply:
It isn't quite that simple, since OnCompletion is in the middle of dialog execution.

The Form is completed here though, so this will already clear the dialog's state.